reflex/app.py

```diff
@@ -1146,7 +1146,7 @@ async def on_event(self, sid, data):
         }
 
         # Get the client IP
-        client_ip = environ["REMOTE_ADDR"]
+        client_ip = environ["asgi.scope"]["client"][0]
```

Comment:
Do we want to add any `get` or `try/catch` here, or will these key lookups always work? We should avoid crashing outside the event loop is my concern

Reply:
yeah i can try/catch it. i didn't because we didn't before, but this is multiple levels of derefs, so more could go wrong